test/index.js

```diff
@@ -388,6 +388,21 @@ describe('Bigtable', function() {
       bigtable.createInstance(INSTANCE_NAME, options, assert.ifError);
     });
 
+    it('should respect the type option', function(done) {
+      var options = {type: 'development'};
+      sinon.spy(FakeInstance, 'getTypeType_');
+
+      bigtable.request = function(config) {
+        assert.deepEqual(config.reqOpts.instance.type, 2);
```

Comment:
This line tests functionality of `Instance.getTypeType_`. We need another test block for the new `Instance.getTypeType_` function to make sure that works as intended.

Reply:
Added a test in https://github.com/googleapis/nodejs-bigtable/pull/110/files#diff-c5fd52f715b6d5d9a356e4b76caf7511